core/src/main/scala/cats/ContravariantMonoidal.scala

```diff
@@ -11,12 +11,22 @@ import simulacrum.typeclass
  * Based on ekmett's contravariant library:
  * https://hackage.haskell.org/package/contravariant-1.4/docs/Data-Functor-Contravariant-Divisible.html
  */
-@typeclass trait ContravariantMonoidal[F[_]] extends ContravariantSemigroupal[F] { self =>
+@typeclass trait ContravariantMonoidal[F[_]] extends ContravariantSemigroupal[F] with InvariantMonoidal[F] {
   /**
-   * `unit` produces an instance of `F` for any type `A`
+   * `conquer` produces an instance of `F` for any type `A`
    * that is trivial with respect to `contramap2` along
    * the diagonal
    */
-  def unit[A]: F[A]
+  def conquer[A]: F[A] = contramap(unit)(_ => ())
```

Comment:
I don't want to bikeshed, but I'm not crazy about `conquer` as a name. I'm guessing it was just a cute name in the context of `Divisible` (`divide`/`conquer`)? Or is there a more meaningful reason behind the name?

Throwing out a few names, none of which I like all that much: `trivial`, `emptyK`, `pureEmpty`

Reply:
I like trivial!